Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -123,6 +130,13 @@ open class NavigationMapView: UIView {
      Most recent user location, which is used to place `UserCourseView`.
      */
     var mostRecentUserCourseViewLocation: CLLocation?
+
+    @objc dynamic public var routeDurationAnnotationSelectedColor: UIColor = .selectedRouteDurationAnnotationColor
+    @objc dynamic public var routeDurationAnnotationColor: UIColor = .routeDurationAnnotationColor
+    @objc dynamic public var routeDurationAnnotationSelectedTextColor: UIColor = .selectedRouteDurationAnnotationTextColor
+    @objc dynamic public var routeDurationAnnotationTextColor: UIColor = .routeDurationAnnotationTextColor
+    @objc dynamic public var routeDurationAnnotationFontName: [String] = ["DIN Pro Medium", "Noto Sans CJK JP Medium", "Arial Unicode MS Regular"]
```

Comment:
I still think a better _default_ behavior should be to match the style’s existing road label fontstack, even if this UIAppearance option remains as an override. But `routeDurationAnnotationFontName` is not new, so this can be tail work.

Reply:
I'm not totally convinced, but I do think you have a good point in that not picking up the map style's font stack removes the ability of the map designer from changing this without help from engineering.

I wrote up https://github.com/mapbox/mapbox-navigation-ios/issues/2994 to track this later change.